_layouts/default_without_photo.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
```

Comment:
I made the refactoring. Now you can show and hide the avatar using the "avatar" variable.